src/SetZipper.elm

```diff
@@ -0,0 +1,48 @@
+module SetZipper exposing (Model, RankedTeam, TeamRankings, createZipper)
```

Comment:
This module seems to be just a shorter (and incomplete) version of `Ranking.elm`, so I'm going to skip it.